script/Ciel/Day 14/14_01B_02_CIEL14B_3.txt

```diff
@@ -111,107 +113,133 @@ What is that?!
+%{/i}
 }
 [sha:e6b0a68d16497faed433eb480d4ad595e3be2094]{
 -- Page 97, Offset 39686.
 -- 神は足元に跪く者にのみ、真実の一端を見せるものだ』
-God will only reveal a fraction of the truth to those who kneel before his feet."
+%{i}
+Only those who kneel at the foot of God will be granted a glimpse of the truth."
+%{/i}
 }
 [sha:04fbcf4a61753a0f8cfc4defbebbffbb31d186f4]{
 -- Page 98, Offset 39687.
 -- ロアの言葉が思考の表面をかすめていく。
-Roa's words brushes against the surface of my thoughts.
+// Reading this as like 'in one ear and out the other', or that he's not super paying attention?
+Roa's words skim across the surface of my thoughts.
 }
 [sha:20ed1c23f80fdd4ce5627c888df5d99a2a530c07]{
 -- Page 98, Offset 39688.
 -- アレが―――アルクェイド？
-Is that―――Arcueid?
+Is that thing... Arcueid?
 }
 [sha:54f77d5e3b206aabe07e5cd631a6d781eb84281a]{
 -- Page 98, Offset 39689.
 -- 異様なんてものじゃない。
-This isn't something that's simply bizzare.
+This goes beyond simply being strange.
 }
 [sha:25024a344e3de0047d932b6856ca60eb97975610]{
 -- Page 98, Offset 39690.
 -- 頭部と思わしき部位は、この街で一番の高層ビルであるハイアットタワーより高い。
-The part that appears to be the head is taller than the Hyatt Tower, the tallest building in the city.
+// Sponsored??
+The part that appears to be the head looms above the Hyatt Tower, the tallest building in the city.
 }
 [sha:5174b36507e95e3e42aefd1562f37c94c6a30177]{
 -- Page 99, Offset 39691.
 -- あのビルは100メートルという触れ込みだった。
-That building is said to be 100 meters high.
+That building is supposedly a hundred meters tall―――
 }
 [sha:836ca2529c4202e8db33a4f06d32b9346218738f]{
 -- Page 99, Offset 39692.
 -- じゃあ、あれは100メートル以上の巨人という事か―――？
-In that case, does that mean this giant is even higher than that―――?
+So does that mean this giant is even taller than %{i}that%{/i}...?
 }
 [sha:69f341fb48d3d3239e28289ceec722655e0cf0c6]{
 -- Page 100, Offset 39693.
 -- 『そうだ。あの光の中心に彼女の意識はあるのだろう。
-"Exactly. Her consciousness must be located at the center of that light.
+%{i}
+"You are correct.
+ Her consciousness is likely located at the center of that light.
+%{/i}
 }
 [sha:f7f8a5f785d7ac444654cdf016ed31118c13fb86]{
 -- Page 100, Offset 39694. Mods: @k, @e.
 -- 肉体という壁がない為、その有り余る魔力量のまま
-Because there is no body to act as a barrier, the abundant mass of magical energy
+%{i}
+That overabundance of magical energy has no body to contain it―――
+%{/i}
 }
 [sha:71fbd94ee9e3fc6a992d116e3b3d5dac7eb8c6df]{
 -- Page 100, Offset 39695.
 -- “彼女が思い描く自身のイメージ”が投影・拡散されている。
-projects and scatters the image of what she imagines herself to be.
+%{i}
+As such, it projects the image of what she imagines herself to be.
+%{/i}
 }
 [sha:c2d7fcf645adcf8bd29a0f81c06b25495d8e938b]{
 -- Page 100, Offset 39696. Mods: @k, @e.
 -- あれは既に、おまえの知る
-That thing is no longer
+%{i}
+By this point―――
+%{/i}
 }
 [sha:5a2fa92a77dbe6680101c3703e9ef41964d7a356]{
 -- Page 100, Offset 39697.
 -- アルクェイド・ブリュンスタッドとは別のものだ』
-the same Arcueid Brunestud that you know."
+%{i}
+That thing is no longer the same Arcueid Brunestud that you knew."
+%{/i}
```

Comment:
I am strongly against broken sentences like this. Either we keep it choppy with ――― to preserve the intent, or I mess with the scripting to make it all render as one line.

Reply:
As long as the line doesn't linebreak, I really don't see the harm. The abundance of the dashes is much more distracting than it would be to split it like this